blockchain/staking/run_a_validator.md

```diff
@@ -4,13 +4,23 @@
 
 This guide will walk you through the process of setting up a new validator on Liberland Chain.
 
+[You can follow along a video here](https://www.youtube.com/watch?v=dV0fK1eqSZY)
+
 Note:
 * *DO NOT* expose your validators RPC ports (9933 and 9944 by default) publicly on the internet. Treat validators with more care than full nodes.
 * You *do not* need to a run a validator to earn LLD staking rewards. As a user you can simply become a nominator and stake your coins without needing to run a validator.
 
 ## Requirements
 
 * ~10 GB of free disk space
+* 2GB of RAM, but swap is ok, see [How to add swap](https://www.cloudsigma.com/adding-swap-space-on-ubuntu-20-04-a-tutorial/) if you have less than 2GB of RAM
+* Liberland wallet with at least 200 LLD. [Ask to be onboarded](https://matrix.to/#/#liberland-node:matrix.org)
+* [PolkadotJs wallet](https://polkadot.js.org/extension/)
+
+## Get wallet
+Go to [PolkadotJs wallet](https://polkadot.js.org/extension/)
+Download extension
+Use the extension and create a new wallet. Save the mnemonic key somewhere safe. If you lose the mnemonic key, your account is lost.
```

Comment:
These render as a single line. Add * or numbers before each line to make it a list.